interfaces/builtin/netlink_audit.go

```diff
@@ -51,14 +59,33 @@ capability audit_read,
 capability audit_write,
 `
 
+type netlinkAuditInterface struct {
+	commonInterface
+}
+
+func (iface *netlinkAuditInterface) BeforeConnectPlug(plug *interfaces.ConnectedPlug) error {
+	features, err := apparmor_sandbox.ParserFeatures()
+	if err != nil {
+		return err
+	}
+
+	if !strutil.ListContains(features, "audit_read") {
```

Comment:
Doesn't this need a corresponding change to `probeParserFeatures()` in `sandbox/apparmor/apparmor.go` to probe for this feature and add it to the parser features, otherwise we will never connect the plug as the feature never exists? Also we seem to name features with hyphens rather than underscores so this should probably be named `audit-read` instead.

Reply:
good catch! I actually had just assumed this was already added there, even though I literally just added code there like a month ago :facepalm: /me hides